src/ui/RealmBrowser/focus.ts

```diff
@@ -47,7 +48,7 @@ export interface IClassFocus extends IFocus {
 
 interface IBaseListFocus extends IFocus {
   kind: 'list';
-  parent: Realm.Object;
+  parent: Realm.Object & { [key: string]: any };
```

Comment:
I wonder if this would be a good use of the `unknown` type instead?
```suggestion
  parent: Realm.Object & { [key: string]: unknown };
```